etc/di.xml

```diff
@@ -79,7 +83,8 @@
     </type>
     <type name="Bazaarvoice\Connector\Console\Command\Purchase">
         <arguments>
-            <argument name="purchaseFeed" xsi:type="object">Bazaarvoice\Connector\Model\Feed\PurchaseFeed\Proxy</argument>
+            <argument name="purchaseFeed" xsi:type="object">Bazaarvoice\Connector\Model\Feed\PurchaseFeed\Proxy
```

Comment:
There are inconsistencies on how the content is incapsulated in the tags.

Reply:
Fixed.